bin/update

```diff
@@ -1,48 +1,54 @@
 #!/usr/bin/env ruby
 require 'pathname'
-require 'English'
```

Comment:
👍  LGTM. The functionality of reporting the command that failed is still there and some lines of code got deleted! Bonus!